### ISSUE TITLE
Kemohno needs to be invited to private channels to be used

### ISSUE THREAD

**Issue by shawnrc** [shawnrc/kemohno#20]
This is, of course, less than ideal, but may be solvable. Might just need to switch from hitting `chat.postMessage` to just replying to the `response_url`, assuming that url still allows setting profile images and names.

If it can't do the user spoofing, then the approach might be to check "if the channel exists" to check if it's a private channel, and if the channel is private, either respond as Kemohno or let the calling user know that it needs to be invited to the channel to mimic lads.

**Comment by shawnrc** [comment]
```
ERROR  16:37:48.798 [Thread-11]        me.shawnrc.kemohno.SlackClient   - ok=false error=channel_not_found
--
  | Exception in thread "Thread-11" java.lang.Exception: bad call to chat.postMessage
  | at me.shawnrc.kemohno.SlackClient$Companion$errorHandler$1.invoke(SlackClient.kt:80)
  | at me.shawnrc.kemohno.SlackClient$Companion$errorHandler$1.invoke(SlackClient.kt:64)
  | at khttp.async$Companion$request$3.invoke(KHttp.kt:113)
  | at khttp.async$Companion$request$3.invoke(KHttp.kt:72)
  | at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:18)
```

**Comment by shawnrc** [comment]
resolved by eb772ff
